frontend/src/components/Container.vue

```diff
@@ -81,6 +83,42 @@ export default class ContainerVue extends Vue {
         }
     }
 
+    #gastspieler &__header {

```

Comment:
This is unacceptable. We need a solution which works for all container with a custom header color.

Reply:
something darken? idk. how to determine when to switch from dark to light?

Reply:
no solution possible with (s)css only imo

Reply:
sth like https://medium.com/better-programming/generate-contrasting-text-for-your-random-background-color-ac302dc87b4 would be needed for a generic solution supporting all colors.

Reply:
found no solution for quotation fill btw

Reply:
Should be fixed in 
b367efa